README.md

```diff
@@ -9,6 +9,14 @@
 
 ---
 ## Work Experience
+
+### Platform at Poggio Labs (Feb 2022 - _Present_)
+*San Francisco, CA (and partially remote)*
```

Comment:
Poggio is [fully remote](https://boards.greenhouse.io/poggiolabs/jobs/4147581004), otherwise LGTM. 🚀 :shipit: